### DISCUSSION TITLE
English models' Accuracy Evaluation values

### DISCUSSION THREAD

**Discussion by ojo4f3** [explosion/spaCy#13185] [Language Support]
Hello!

I have a question regarding the English model stats displayed on the spaCy website: [https://spacy.io/models/en](https://spacy.io/models/en). The Accuracy Evaluation tables for the `en_core_web_sm`, `en_core_web_md`, and `en_core_web_lg` models are exactly the same (although the `en_core_web_lg` model's Named entities (recall) value is 0.86 instead of 0.85 like the others). The `en_core_web_trf` model's values are unique.

I understand these values may be correct but wanted to make sure since I'd assume that the larger models would have better results.

Thanks,

Colton

**Comment by adrianeboyd** [comment]
English has enough training data that larger models and vectors don't make a huge difference. All the raw evaluation numbers are here (the website tables are dynamically directly generated from this data):

https://github.com/explosion/spacy-models/tree/master/meta
